src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -469,9 +480,12 @@ public async Task<AuthenticationResponse> AuthenticateAsync(AuthenticationOption
                     introspectResponse);
         }
 
-        private async Task<AuthenticationResponse> AuthenticateWithPasswordAsync(AuthenticationOptions authenticationOptions, CancellationToken cancellationToken = default)
+        private async Task<AuthenticationResponse> AuthenticateWithPasswordAsync(AuthenticationOptions authenticationOptions, CancellationToken cancellationToken = default, string recoveryToken = null)
         {
-            var idxContext = await InteractAsync(cancellationToken: cancellationToken, activationToken: authenticationOptions.ActivationToken);
+            var idxContext = await InteractAsync(
+                cancellationToken: cancellationToken,
+                activationToken: authenticationOptions.ActivationToken,
+                recoveryToken: recoveryToken);

```

Comment:
If someone is authenticating with a password then recoveryToken shouldn't be necessary.